opentech/public/news/views.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.syndication.views import Feed
+
+from opentech.public.news.models import NewsPage, NewsType
+
+
+class NewsFeed(Feed):
+    link = "https://www.opentech.fund/news/"
+    title = "OTF – News, updates, and announcements"
+    description = "News, updates, and announcements from The Open Technology Fund"
+
+    def items(self):
+        return NewsPage.objects.live().order_by('-first_published_at')[:20]
+
+    def item_title(self, item):
+        return item.title
+
+    def item_description(self, item):
+        return item.body
```

Comment:
NewsPage's body field is a streamfield for content. Using `introduction` field will be good here as description is for item synopsis.

Reply:
I see your point but introduction field is not used a lot in practice and I want a full content rss feed. Tested the feed in my feed reader and it seems to work as it should with body.